compiler/rustc_smir/src/rustc_smir/mod.rs

```diff
@@ -54,6 +54,36 @@ impl<'tcx> Context for Tables<'tcx> {
         self.tcx.sess.source_map().span_to_diagnostic_string(self[span])
     }
 
+    fn get_filename(&self, span: &Span) -> Filename {
```

Comment:
Should we return local or remapped instead? It is easier to extract the filename from the complete path, but no the other way around.